### PR TITLE
Glimpse: add usp support, update params and fpd setup

### PR DESCRIPTION
**Pull request by eddieyu1998**
# Description

- Add usp support
- Remove demand and keywords bidder params
- Update example on setting up adapter with first party data
- 
## 🏷 Type of documentation

- [ ] new bid adapter
- [x] update bid adapter
- [x] new feature
- [ ] text edit only (wording, typos)
- [ ] bugfix (code examples)
- [x] new examples

## 📋 Checklist

- [ ] Related pull requests in prebid.js or server are linked

## Related Issues/PRs

https://github.com/glimpse-protocol/prebid/issues/1
https://github.com/glimpse-protocol/Prebid.js/pull/12

### CODE REVIEW

File: dev-docs/bidders/glimpse.md [hunk 1]
```diff
@@ -96,34 +91,45 @@ const units = [
...
-        keywords: "sports, basketball",
-        ext: {
-          data: {
-            title: "The Premier League",
+  pbjs.setBidderConfig({
+    bidders: ["glimpse"],
+    config: {
+      ortb2: {
+        site: {
+          keywords: "business, finance, crypto", // site specific keywords, separated by comma
```
Comment (tim-hm) [inline dev-docs/bidders/glimpse.md:105]: these should be in an array

**Review by tim-hm — CHANGES_REQUESTED** [review]
a few tweaks please 🙏

File: dev-docs/bidders/glimpse.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ description: Glimpse Protocol Bid Adapter
 biddercode: glimpse
 pbjs: true
 gdpr_supported: true
+usp_supported: true
 deals_supported: true
```
[the remark this answers is not in the document]
Comment (eddieyu1998) [reply dev-docs/bidders/glimpse.md:9]: https://docs.prebid.org/adops/deals.html#enable-deals-in-prebid

it feels to me it's likely what we are doing right now?

File: dev-docs/bidders/glimpse.md [hunk 3]
```diff
@@ -63,11 +64,9 @@ Coming soon.
...
-| `placementId` | Required | string              | A unique identifier associated with a publisher and ad unit, provided by the Glimpse Publisher Portal when registering a unit | 'glimpse-placement-id' |
-| `demand`      | Optional | string              | Target demand source (defaults to 'glimpse')                                                                                  | 'xandr'                |
-| `keywords`    | Optional | Record<string, any> | Audience targeting data (applies to the single ad unit)                                                                       | { sticky: true }       |
+| Name          | Scope    | Type   | Description                                                                                                                   | Example                |
+| ------------- | -------- | ------ | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------- |
+| `placementId` | Required | string | A unique identifier associated with a publisher and ad unit, provided by the Glimpse Publisher Portal when registering a unit | 'glimpse-placement-id' |
```
[the remark this answers is not in the document]
Comment (eddieyu1998) [reply dev-docs/bidders/glimpse.md:69]: we can, I'll update the adapter and test accordingly later

**Review by tim-hm — APPROVED** [review]
I made a few tweaks. Please give it a sanity check and then go ahead and merge 🙏

**Pull request merged**